scripts/globals/mobskills/Cyclonic_Turmoil.lua

```diff
@@ -1,5 +1,32 @@
 ---------------------------------------------
 -- Cyclonic_Turmoil
---
---
+-- Deals Wind damage in an area of effect. Additional effect: Knockback & Dispel
+-- Notes: Dispels multiple buffs.  Wipes shadows.
 ---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+    local dis1 = target:dispelStatusEffect();
+    local dis2 = target:dispelStatusEffect();
+
+
+    if (dis1 ~= EFFECT_NONE and dis2 ~= EFFECT_NONE) then
+        skill:setMsg(MSG_DISAPPEAR_NUM);
+        return 2;
+    elseif (dis1 ~= EFFECT_NONE or dis2 ~= EFFECT_NONE) then
+        -- dispelled only one
+        skill:setMsg(MSG_DISAPPEAR_NUM);
+        return 1;
+    else
+        skill:setMsg(MSG_NO_EFFECT); -- no effect
+    end
+
+    local dmgmod = 1;
+    local info = MobMagicalMove(mob,target,skill,mob:getWeaponDmg()*2.8,ELE_WIND,dmgmod,TP_NO_EFFECT);
+    local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_MAGICAL,MOBPARAM_WIND,MOBPARAM_WIPE_SHADOWS);
+    target:delHP(dmg);
+    return dmg;
+end;
```

Comment:
This script is just plain wrong. Its the exact script I have seen before on a another server that was crashing clients from it. Did you by chance get this from a forum thread? If it was form an existing skill script, that one is busted too. You set the no effect msg but then return dmg. What you want in that last scenario is to either return nothing or to use a damage message, then return the damage (depending on which message retail actually displays).